### PR TITLE
docs: Make it more clear that synced users have `User` role

### LINKED ISSUE #5906: Better error when attempting to log in as user to portal
A common workflow is:

- User signs up using email and is admin
- User connects IdP
- Synced user doesn't have admin
- User tries to login via IdP and is greeted with "Please login from a client"

Fix:

- better error
- update docs
- add UI hints / notes when creating the connector

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes #5906

### CODE REVIEW

File: website/src/app/kb/authenticate/entra/readme.mdx [hunk 1]
```diff
@@ -425,17 +425,28 @@ Go back to the setup page in the Firezone admin portal, ensure all fields are
 filled out, and click **Connect Identity Provider**.
 
 <Alert color="warning">
-  If directory sync is enabled, all users and groups are synced by default. You
-  can limit which users and groups are synced in the [Enteprise
-  Applications](https://portal.azure.com/#view/Microsoft_AAD_IAM/StartboardApplicationsMenuBlade/~/AppAppsPreview)
-  section of the Azure portal. See the [Microsoft
-  documentation](https://learn.microsoft.com/en-us/entra/identity-platform/howto-restrict-your-app-to-a-set-of-users)
-  for more information.
+
```
Comment (jamilbk) [inline website/src/app/kb/authenticate/entra/readme.mdx:428]: just formatting changes

File: elixir/apps/web/lib/web/live/settings/identity_providers/microsoft_entra/show.ex [hunk 2]
```diff
@@ -84,12 +84,16 @@ defmodule Web.Settings.IdentityProviders.MicrosoftEntra.Show do
...
       </:action>
       <:help>
-        Directory sync is enabled for this provider. Users and groups will be synced every 10
-        minutes on average, but could take longer for very large organizations.
-        <.website_link path="/kb/authenticate/directory-sync">
-          Read more
-        </.website_link>
-        about directory sync.
+        <p>
+          Directory sync is enabled for this provider. Users and groups will be synced every 10
```
Comment (bmanifold) [inline elixir/apps/web/lib/web/live/settings/identity_providers/microsoft_entra/show.ex]: I noticed you changed some of the other provider messages from `every 10 minutes` to `every few minutes`, were some of them left at 10 min intentionally?

Comment (jamilbk) [reply elixir/apps/web/lib/web/live/settings/identity_providers/microsoft_entra/show.ex]: good catch, updated the rest

**Review by bmanifold — APPROVED** [review]
LGTM

**Pull request merged**